tests/download.spec.ts

```diff
@@ -31,6 +31,16 @@ it.describe('download event', () => {
       res.setHeader('Content-Disposition', 'attachment; filename=file.txt');
       res.end(`Hello world`);
     });
+    server.setRoute('/downloadWithDelay', (req, res) => {
+      res.setHeader('Content-Type', 'application/octet-stream');
+      res.setHeader('Content-Disposition', 'attachment; filename=file.txt');
+      // Chromium requires a large enough payload to trigger the download event soon enough
+      res.write('a'.repeat(4096));
+      res.write('foo');
+      setTimeout(() => {
```

Comment:
just flush (see `cork()/uncork()` methods) the response bytes and remove setTimeout altogether so that the transmission never ends (it is cancelled anyway).